pyo3-macros-backend/src/pymethod.rs

```diff
@@ -491,15 +497,31 @@ fn impl_arg_param(
     };
 
     if spec.is_args(&name) {
-        return quote! {
-            let #arg_name = <#ty as pyo3::FromPyObject>::extract(_args.as_ref())
-                .map_err(#transform_error)?;
+        return if arg.optional.is_some() {
```

Comment:
Does this mean we'll start supporting `args: Option<&PyTuple>`?
Then let's add a test and document for that.

Reply:
Actually it's still not possible, because I can't figure out the right macro invocation to get `Option<&PyTuple>` and `Option<Vec>` to work correctly (we currently support e.g. both `&PyTuple` and `Vec`.

I'll add a check that the type is _not_ `Option<_>` for now.